internal/app/encryption.go

```diff
@@ -113,7 +119,11 @@ func Encrypt(dataStr string, passphrase string) []byte {
 // Decrypt ...
 func Decrypt(dataStr string, passphrase string) []byte {
 	dataByte := []byte(dataStr)
-	key := []byte(CreateHash(passphrase))
+	hash, err := CreateHash(passphrase)
+	if err != nil {
+		panic(err.Error())
```

Comment:
Please do not use panic anywhere in the application.

Reply:
I added panic just because there is already a usage of a panic a couple of lines below. Should I delete those panics as well?

Reply:
Yes, please.

Reply:
Deleted pacic statements